packages/react-button/src/components/Button/useButtonState.ts

```diff
@@ -32,5 +32,6 @@ export const useButtonState = (draftState: ButtonState) => {
     }
   }
 
-  draftState.disabled = draftState['aria-disabled'] = draftState.disabled || draftState.loading;
+  draftState['aria-disabled'] = draftState.disabled || draftState.loading;
```

Comment:
in react-northstar, loading button has also aria-disabled by default, but I was questioning that in my mind couple of times - because we are mixing two concepts. I am leaning towards separating that - if the consumer wants to use loading, he will express that with `loading` prop, if the intention is to disable it as well, he should use `disabled` or `disabledFocusable`. This way we will be doing less assumptions and I think it will become more understandable.

Reply:
I'd second that as less assumptions and more understandable components is always better in my book. `loading` putting `disabled` by default always seemed a little odd to me anyway. I'll update the PR with that change.